fr/development/routing.rst

```diff
@@ -1536,7 +1536,7 @@ Les méthodes ci-dessus acceptent également un tableau d'options comme deuxièm
     // Pour lequel le timestamp correspond à la date de dernière modification du fichier
     $img = Asset::url('logo.png', ['timestamp' => true]);
 
-Pour générer des URL de ressources pour les fichiers dans les plugins, utilisez :term:`syntaxe de plugin`::
+Pour générer des URL de ressources pour les fichiers dans les plugins, utilisez :term:`plugin syntax`::
```

Comment:
This should stay in french as the glossary term is also french.

Reply:
Fixed in 921c695